task/make.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function make(a) {
+  const newArr = [a];
+  function f(b) {
+    if (typeof (b) === 'function') {
+      return newArr.reduce(b);
+    }
+    const args = [...arguments];
```

Comment:
ESLint highlight the error here... We should avoid directly accessing the `arguments` pseudo-array.
I assume that you probably just didn't find out how it could be done differently. So here are the tips:

**1)** the ESLint error says: `[eslint] Use the rest parameters instead of 'arguments'.`
You can read about there `rest parameters` here: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters

**2)** If, for some special reason, you need to violate the ESlint rule in a file, let's say, to use `arguments` pseudo-array, you can **disable** a specific rule for the a file or for an entire project.
https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments
https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments
**Remember** that this configurations should be used only for specific cases, avoid using them.